### PR TITLE
Feature/wizard part 2

### PR DESCRIPTION
**Pull request by jasperswart**
## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: app/meldingen/incident/containers/IncidentContainer/reducer.js [hunk 1]
```diff
@@ -17,6 +17,19 @@ import {
...
+    //   address: 'Pieter Nieuwlandstraat 119'
+    // },
+    // subcategory: 'Overlast op het water - snel varen',
+    // category: 'Overlast op het water',
+    // description: 'snel boot',
+    // incident_date: 'now',
+    // phone: '020654321',
+    // email: 'a@b.nl',
+    // extra_boten_snelheid_rondvaartboot: 'Ja',
+    // extra_boten_snelheid_rederij: 'Aemstelland',
```
Comment (aditudorache) [inline app/meldingen/incident/containers/IncidentContainer/reducer.js]: comments

File: app/meldingen/incident/containers/IncidentContainer/saga.js [hunk 2]
```diff
@@ -10,46 +10,112 @@ import {
...
 export default function* watchIncidentContainerSaga() {
   yield [
-    yield takeLatest(CREATE_INCIDENT, createIncident),
-    yield takeLatest(GET_CLASSIFICATION, getClassification)
+    yield takeLatest(GET_CLASSIFICATION, getClassification),
+    yield takeLatest(CREATE_INCIDENT, createIncident)
   ];
 }
+
+/*
```
Comment (aditudorache) [inline app/meldingen/incident/containers/IncidentContainer/saga.js]: unused text

File: app/meldingen/incident/definitions/wizard-step-vulaan.js [hunk 3]
```diff
@@ -41,16 +42,78 @@ export default {
...
           label: 'Wat is de naam van de rederij? (niet verplicht)',
           subtitle: 'Als u begint met typen verschijnt vanzelf een lijst met rederijen',
-          values: [
-            'Geen',
-            'Lovers',
-            'Aemstelland',
-            'Staets'
-          ],
+          path: 'text_extra',
+          values: {
```
Comment (aditudorache) [inline app/meldingen/incident/definitions/wizard-step-vulaan.js]: can this be done as an import from a definition? see ../admin/definitions

File: app/meldingen/incident/definitions/wizard.js [hunk 4]
```diff
@@ -233,12 +237,144 @@ export default {
...
     form: {
       controls: {
-        privacy_text: {
+        text_default: {
+          meta: {
+            cols: 6,
+            label: 'Wat doen we met uw melding?',
+            type: 'bedankt',
+            if: {
+              subcategory: [
```
Comment (aditudorache) [inline app/meldingen/incident/definitions/wizard.js]: move this to the definitions

File: app/meldingen/incident/services/map-controls-to-params/index.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  forEach(map, (item) => {
+    set(params, item.path, item.value);
+  });
+
+  return params;
+};
+
+export default mapControlsToParams;
+
+/*
```
Comment (aditudorache) [inline app/meldingen/incident/services/map-controls-to-params/index.js]: delete this

File: app/meldingen/incident/containers/IncidentContainer/reducer.js (hunk 1 above)
Comment (jasperswart) [reply app/meldingen/incident/containers/IncidentContainer/reducer.js]: die laat ik nog even staan voor testen

File: app/meldingen/incident/services/map-controls-to-params/index.js (hunk 5 above)
Comment (jasperswart) [reply app/meldingen/incident/services/map-controls-to-params/index.js]: gedaan

File: app/meldingen/incident/definitions/wizard-step-vulaan.js (hunk 3 above)
Comment (jasperswart) [reply app/meldingen/incident/definitions/wizard-step-vulaan.js]: we can do this later after mvp

File: app/meldingen/incident/definitions/wizard.js (hunk 4 above)
Comment (jasperswart) [reply app/meldingen/incident/definitions/wizard.js]: we can do this later after mvp

**Pull request merged**